app/components/CrashLocationBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import Alert from "react-bootstrap/Alert";
+
+interface CrashLocationBannerProps {
+  privateDriveFlag: boolean | null;
```

Comment:
would you add a comment here so that it shows up in component's type hints? E.g. "If the crash occurred on a private drive".

Reply:
done! also I learned its very particular of it being a comment with `/**` to start, `/*` wont cut it

Reply:
basically it has to be a JSDoc annotation and not a comment